packages/data-portal-table/src/lib/helpers.tsx

```diff
@@ -3,9 +3,10 @@ import * as React from 'react';
 export function getColumnKey(col: {
     id?: string | number;
     name: string | number | React.ReactNode;
+    dropdownName?: string;
 }): string {
     // if no id exists, just use name for key
-    return (col.id || col.name || '').toString();
+    return (col.id || col.dropdownName || col.name || '').toString();
```

Comment:
we should be able to use `id` instead of introducing a new `dropdownName`

Reply:
ah gotcha, thank you!